salt/metalk8s/orchestrate/bootstrap/init.sls

```diff
@@ -170,3 +172,11 @@ Deploy MetalK8s UI:
     - saltenv: {{ saltenv }}
     - require:
       - salt: Precheck for MetalK8s UI
+
+Store MetalK8s version in annotations:
+  metalk8s_kubernetes.namespace_annotation_present:
+    - name: "kube-system"
+    - kubeconfig: {{ kubeconfig }}
+    - context: {{ context }}
+    - annotation_key: "metalk8s.scality.com/version"
+    - annotation_value: {{ version }}
```

Comment:
Make sure to quote this value. When `version` would ever be e.g. `2.0`, then this will be a YAML float, and K8s doesn't like floats as annotation values.

Reply:
Well noted!